eng/common/pipelines/templates/steps/verify-agent-os.yml

```diff
@@ -14,3 +14,5 @@ steps:
       filePath: ${{ parameters.ScriptDirectory }}/Verify-AgentOS.ps1
       arguments: >
         -AgentImage "${{ parameters.AgentImage }}"
+
+  - template: /eng/common/pipelines/templates/steps/bypass-local-dns.yml
```

Comment:
Is it safe to assume that if a pipeline includes this more then once it is a noop instead of an error?

Reply:
It should be.  The template just runs a single command to create a symlink, and since it includes `-f` (meaning `force`) it should overwrite any existing file/symlink.

https://github.com/Azure/azure-sdk-tools/blob/89a4731f53f29b240884c2c9ff34068eaf106241/eng/common/pipelines/templates/steps/bypass-local-dns.yml#L4

Reply:
After this is merged, I will be removing the redundant calls to `bypass-local-dns.yml` from each language repo.